.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  vvgo:
+    runs-on: ubuntu-latest
+    steps:
+      - name: deploy
+        run: |
+          IMAGE_ID=docker.pkg.github.com/${{ github.repository }}/$IMAGE_NAME
+
+          # Change all uppercase to lowercase
+          IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+
+          # Strip git ref prefix from version
+          VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+
+          # Strip "v" prefix from tag name
+          [[ "${{ github.ref }}" == "refs/tags/"* ]] && VERSION=$(echo $VERSION | sed -e 's/^v//')
+
+          # Use Docker `latest` tag convention
+          [ "$VERSION" == "master" ] && VERSION=latest
```

Comment:
Ha I just realized that this would always tag version latest.

Reply:
It only tags version latest if the branch is master.

... Which, given that we're only releasing master, will always be true.